common/annotations/en.xml

```diff
@@ -718,532 +718,548 @@ annotations.
 		<annotation cp="♭" type="tts">flat</annotation>
 		<annotation cp="♯">dièse | diesis | music | note | sharp</annotation>
 		<annotation cp="♯" type="tts">sharp</annotation>
-		<annotation cp="😀">face | grin | grinning face</annotation>
+		<!-- start hand-generated lines from E16 -->
+		<annotation cp="🪉">music | instrument | love | cupid | orchestra</annotation>
+		<annotation cp="🪉" type="tts">harp</annotation>
+		<annotation cp="🪏">dig | spade | hole | scoop</annotation>
+		<annotation cp="🪏" type="tts">shovel</annotation>
+		<annotation cp="🪾">winter | barren | drought</annotation>
+		<annotation cp="🪾" type="tts">leafless tree</annotation>
+		<annotation cp="🫆">forensics | identity | safety</annotation>
+		<annotation cp="🫆" type="tts">fingerprint</annotation>
+		<annotation cp="🫜">root | vegetable | garden | beet | turnip</annotation>
+		<annotation cp="🫜" type="tts">root vegetable</annotation>
+		<annotation cp="🫟">paint | holi | spill | stain</annotation>
+		<annotation cp="🫟" type="tts">splatter</annotation>
+		<annotation cp="🫩">tired | sleepy | exhausted</annotation>
+		<annotation cp="🫩" type="tts">face with bags under eyes</annotation>
+		<!-- end hand-generated lines from E16 -->
+		<annotation cp="😀">face | grin | grinning face | cheerful | cheery | happy | laugh | nice | smile | smiling | teeth</annotation>
 		<annotation cp="😀" type="tts">grinning face</annotation>
-		<annotation cp="😃">face | grinning face with big eyes | mouth | open | smile</annotation>
+		<annotation cp="😃">face | grinning face with big eyes | mouth | open | smile | awesome | grin | happy | teeth | smiling | yay | smiling face with open mouth</annotation>
 		<annotation cp="😃" type="tts">grinning face with big eyes</annotation>
-		<annotation cp="😄">eye | face | grinning face with smiling eyes | mouth | open | smile</annotation>
+		<annotation cp="😄">eye | face | grinning face with smiling eyes | mouth | open | smile | grin | happy | laugh | lol | smiling | smiling face with open mouth &amp; smiling eyes</annotation>
```

Comment:
Does &amp make sense from a search perspective?

Reply:
I just removed those. I retained them in 3 cases: annotations for & itself, and for 🔣 (so that it comes up if any of the symbols it shows are typed.

https://github.com/unicode-org/cldr/pull/3719/commits/5faa5460b3d56f6556f937b356fbe25ef6c9cf81